homeassistant/components/cover/somfy.py

```diff
@@ -0,0 +1,106 @@
+"""
```

Comment:
Have a look at https://github.com/home-assistant/architecture/issues/124

you are now required to consolidate all platforms under your component dir

Reply:
Oh, I like this new structure. I've updated my code to be compliant.